device/dell/x86_64-dell_z9100_c2538-r0/Force10-Z9100-C8D48/qos.json.j2

```diff
@@ -0,0 +1,8 @@
+{%- macro generate_port_lists(PORT_ALL) %}
+    {# Generate list of ports #}
+    {% for port_idx in range(0, 128, 2) if port_idx not in [26, 30, 34, 38, 106, 110, 114, 118] %}
+        {% if PORT_ALL.append("Ethernet%d" % (port_idx)) %}{% endif %}
+    {% endfor %}
+{%- endmacro %}
```

Comment:
generate_port_lists() will not be called

Reply:
will remove